### PR TITLE
Each student details

### PR DESCRIPTION
**Pull request by Richmond-Andoh**

### CODE REVIEW

File: CorsaSchoolManagementSystem/Views/studentDetails.php [hunk 1]
```diff
@@ -23,139 +29,303 @@
...
-                <li> <strong>TEL. NUMBER :</strong>  03423231322</li>  
-                <li> <strong>OCCUPATION :</strong> 03423231322</li>  
-                <li> <strong>HOMETOWN :</strong> Sefwi Bekwai </li>  
-            </ul>
+    <!-- bills payment -->
+
+    <!-- Button trigger modal -->
+  <div class="container" style="box-shadow: 0 15px 20px 0 rgba(0, 0, 0, 0.1), 0 6px 15px 0 rgba(0, 0, 0, 0.14);">
+    <button type="button" class="btn btn-primary mt-5 mb-2 mx-5" data-bs-toggle="modal" data-bs-target="#staticBackdrop" style="box-shadow: 0 15px 20px 0 rgba(0, 0, 0, 0.1), 0 6px 15px 0 rgba(0, 0, 0, 0.14);">
+      Payment Fees
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Views/studentDetails.php]: Change `Payment Fees` to  `Add Fee Payment`

File: CorsaSchoolManagementSystem/seeder.php [hunk 2]
```diff
@@ -4,9 +4,9 @@
 $db = new Database();
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/seeder.php:4]: No need to change the seeder file

File: CorsaSchoolManagementSystem/Views/studentDetails.php [hunk 3]
```diff
@@ -23,139 +29,303 @@
...
-                <li> <strong>MOTHER'S NAME :</strong> Mr. Samuel Atikuni </li>  
-                <li> <strong>TEL. NUMBER :</strong>  03423231322</li>  
-                <li> <strong>OCCUPATION :</strong> 03423231322</li>  
-                <li> <strong>HOMETOWN :</strong> Sefwi Bekwai </li>  
-            </ul>
+    <!-- bills payment -->
+
+    <!-- Button trigger modal -->
+  <div class="container" style="box-shadow: 0 15px 20px 0 rgba(0, 0, 0, 0.1), 0 6px 15px 0 rgba(0, 0, 0, 0.14);">
+    <button type="button" class="btn btn-primary mt-5 mb-2 mx-5" data-bs-toggle="modal" data-bs-target="#staticBackdrop" style="box-shadow: 0 15px 20px 0 rgba(0, 0, 0, 0.1), 0 6px 15px 0 rgba(0, 0, 0, 0.14);">
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Views/studentDetails.php]: This button shows Terminal Report Modal...is it dummy information? You can update it to have the right information. 

Screenshot 2022-12-07 at 3 26 15 PM

File: CorsaSchoolManagementSystem/Views/studentDetails.php [hunk 4]
```diff
@@ -23,139 +29,303 @@
...
+        </table>
+      </div>
     </div>
   </div>
-</div>
-</div>
+
+  
+
+  <footer class="bg-primary mt-5">
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Views/studentDetails.php]: Also this is not necessary

**Comment by Richmond-Andoh** [comment]
Alright

**Pull request merged**